integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/testsupport/util/ElasticsearchTckBackendFeatures.java

```diff
@@ -52,8 +51,9 @@ public boolean normalizesStringMissingValues() {
 	public boolean normalizesStringArgumentToWildcardPredicateForAnalyzedStringField() {
 		// In ES 7.7 through 7.11, wildcard predicates on analyzed fields get their pattern normalized,
 		// but that was deemed a bug and fixed in 7.12.2+: https://github.com/elastic/elasticsearch/pull/53127
+		// Apparently the "fix" was also introduced in OpenSearch 2.5.0
 		return isBetween( actualVersion, "elastic:7.7", "elastic:7.12.1" ) ||
-				isOpenSearch( actualVersion );
+				!isLessThan( actualVersion, "opensearch:2.5.0" );
```

Comment:
Judging from the code, `isLessThan( actualVersion, "opensearch:2.5.0" )` would return `false` for any version of Elasticsearch (see `tryOrFalse`), so your change will make `normalizesStringArgumentToWildcardPredicateForAnalyzedStringField` return true for every version of Elasticsearch... not good?

Reply:
hmm I've looked at the other conditions that we have, and it makes me wonder... Maybe we should be more explicit:
```
return condition(
		actualVersion,
		esVersion -> !isLessThan( esVersion, "elastic:6.4.0" ),   // check for ES
		osVersion -> true                                                                 // check for OS
);
```
Something along those lines. WDYT?

Reply:
Ok, but please make it so we don't need the `elastic:` prefix anymore, then.